packages/security/src/decorators/authenticator.decorator.ts

```diff
@@ -1,9 +1,13 @@
 import {AuthenticatorInterface} from "../interfaces/authenticator.interface";
-import {AuthenticatorInstantiationError} from "../errors/authenticator-instantiation.error";
-import {GuardContextInterface} from "../interfaces/guard-context.interface";
 import {AuthenticatorContextInterface} from "../interfaces/authenticator-context.interface";
 import {AuthenticatorDecoratorError} from "../errors/authenticator-decorator.error";
 
+/**
+ * This decorator is to specify the authenticator that should be used to authenticate a request.
```

Comment:
this..  specifies. "is to" doesn't really work.

Reply:
```suggestion
 * This decorator specifies the authenticator that should be used to authenticate a request.
```